src/commands/snapshot.ts

```diff
@@ -0,0 +1,104 @@
+import {flags} from '@oclif/command'
+import Constants from '../services/constants'
+import {StaticSnapshotOptions} from '../services/static-snapshot-options'
+import StaticSnapshotService from '../services/static-snapshot-service'
+import PercyCommand from './percy-command'
+
+export default class Snapshot extends PercyCommand {
+  static description = 'Snapshot a directory of webpages'
+  static hidden = true
+
+  static args = [{
+    name: 'snapshotDirectory',
+    description: 'A path to the directory you would like to snapshot',
+    required: true,
+  }]
+
+  static examples = [
+    '$ percy snapshot _mySite/',
+    '$ percy snapshot _mySite/ --baseUrl "blog/"',
+    '$ percy snapshot _mySite/ --widths "320,780,1280"',
+    '$ percy snapshot _mySite/ --ignore-folders "Tmp,_secrets,node_modules"',
+  ]
+
+  static flags = {
+    'snapshot-capture-regex': flags.string({
+      char: 'c',
+      description: 'Regular expression for matching the files to snapshot.',
+      default: '\.(html|htm)$',
+    }),
+    'ignore-folders': flags.string({
+      char: 'i',
+      description: 'Comma-seperated string of folders to ignore. Ex: Tmp,_secrets,node_modules',
+      default: '',
+    }),
+    'widths': flags.string({
+      char: 'w',
+      description: 'Comma-separated string of rendering widths for snapshots.',
+      default: '1280',
+    }),
+    'baseUrl': flags.string({
```

Comment:
Suggestion to rename to `base-url` to keep consistent with Oclif's conventions.

Reply:
👍